### PR TITLE
Add support to send metadata

### PR DESCRIPTION
**Pull request by virajkarandikar**

### CODE REVIEW

File: riva/clients/nmt/riva_streaming_s2s_client.cc [hunk 1]
```diff
@@ -112,8 +114,10 @@ main(int argc, char** argv)
   str_usage << "           --print_transcripts=<true|false> " << std::endl;
   str_usage << "           --output_filename=<string>" << std::endl;
   str_usage << "           --verbatim_transcripts=<true|false>" << std::endl;
-  str_usage << "           --source_language_code=<bcp 47 language code (such as en-US)>" << std::endl;
-  str_usage << "           --target_language_code=<bcp 47 language code (such as en-US)>" << std::endl;
+  str_usage << "           --source_language_code=<bcp 47 language code (such as en-US)>"
+            << std::endl;
+  str_usage << "           --target_language_code=<bcp 47 language code (such as en-US)>"
+            << std::endl;
```
Comment (rmittal-github) [inline riva/clients/nmt/riva_streaming_s2s_client.cc:120]: add metadata usage string here

**Comment by rmittal-github** [comment]
Looks good

**Review by rmittal-github — APPROVED** [review]

File: riva/clients/nmt/riva_streaming_s2s_client.cc (hunk 1 above)
Comment (virajkarandikar) [reply riva/clients/nmt/riva_streaming_s2s_client.cc:120]: done

**Comment by virajkarandikar** [comment]
Alternate implementation at https://github.com/nvidia-riva/cpp-clients/pull/59/ using GRPC composite call credentials plugin

**Comment by rmittal-github** [comment]
https://github.com/nvidia-riva/cpp-clients/pull/59 is merged, closing this one.

**Pull request closed without merging**